src/config.ts

```diff
@@ -29,6 +30,32 @@ import { chainConfig, publicActionsL1, publicActionsL2 } from "viem/op-stack";
  * TODO:
  * - load walletConnect project id from the environment:
  */
+//
+const JibchainTestnet = defineChain({
+  // ...chainConfig,
+  id: 88991,
+  name: 'JibchainTestnet',
+  nativeCurrency: {
+      decimals: 18,
+      name: 'tJBC',
+      symbol: 'tJBC',
+  },
+  rpcUrls: {
+      default: {
+          http: ['https://rpc.testnet.jibchain.net'],
+          webSocket: [''],
+      },
+  },
+  blockExplorers: {
+      default: { name: 'Explorer', url: 'https://exp.testnet.jibchain.net/' },
+  },
+  contracts: {
+    multicall3: {
+      address: '0xa1a858ad9041B4741e620355a3F96B3c78e70ecE',
+      blockCreated: 32848,
+    },
+  },
+});
```

Comment:
I would be happy to accept a PR that extracted the creation of a parentChain and relied on environment variables for configuration. I will note that there are some pending changes in a draft PR that would need to be taken into account though.